packages/desktop-client/src/components/autocomplete/NewAutocomplete.tsx

```diff
@@ -1,27 +1,47 @@
 import React, { useState } from 'react';
 import Select, {
+  GroupBase,
   Props as SelectProps,
   PropsValue,
   SingleValue,
   SelectInstance,
 } from 'react-select';
 
+import type { CreatableProps } from 'react-select/creatable';
 import CreatableSelect from 'react-select/creatable';
 
 import { NullComponent } from '../common';
 
 import styles from './autocomplete-styles';
 
-type OptionValue = string;
+type OptionValue = {
+  __isNew__?: boolean;
```

Comment:
Why wouldn’t this be just `isNew`?

Reply:
The `__isNew__` is set internally by `react-select` when you create a new autocomplete element. Not controlled by us actually.